src/client/containers/Home/Home.styles.css

```
@@ -0,0 +1,74 @@
+nav {
+  display: none;
+}
+.body {
+  background-color: white;
+}
+
+.home-container-logo-buttons {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 100vh;
+  overflow: auto;
+}
+
+.home-img {
+  margin-top: 5%;
+  display: flex;
+  justify-content: center;
+}
+
+.home-logoAnna {
+  width: 400px;
+  height: 400px;
+}
+.home-buttons {
+  bottom: 20%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.home-buttonSignUp {
```

There are more places in the code where you need the right html class names

```suggestion
.home-button-signup {
```